cedar-policy/src/api.rs

```diff
@@ -379,6 +379,13 @@ impl From<cedar_policy_core::entities::Entities> for Entities {
     }
 }
 
+#[doc(hidden)]
+impl From<Entities> for cedar_policy_core::entities::Entities {
+    fn from(entities: Entities) -> Self {
+        entities.0
+    }
+}
```

Comment:
I think this might be the first instance of a From<API type> for <core type>; we do already have an `AsRef` for this conversion.  But if you do need owned `cedar_policy_core::entities::Entities`, this is more efficient than `.as_ref().clone()`, so seems reasonable.  Should we have similar instances for all other cases where we currently have `AsRef` instances?